client-admin/src/constants.js

```diff
@@ -0,0 +1,8 @@
+export const domainWhitelist = [
+  '^localhost$',
+  '^192\\.168\\.1\\.140$',
+  '^pol\\.is',
+  '.+\\.pol\\.is$',
+  '^xip\\.io$',
+  '.+\\.xip\\.io$'
+]
```

Comment:
This will need a rethink when we start preparing this setup for generalized production.

I think we're safe to defer this for now, since it works on localhost, pol.is domains, and xip.io development, which is really all it's "sanctioned" for rn